c2corg_ui/static/js/utils.js

```diff
@@ -123,6 +123,30 @@ app.utils.formatDate = function(date) {
 
 
 /**
+ * check that the start date if before end date.
+ * Returns an error message if there is an error with dates.
+ * @export
+ * @param {Object} outing
+ * @return {(undefined | string)}
+ */
+app.utils.checkDates = function(outing) {
```

Comment:
Instead of displaying errors when the user select incorrect dates, would it be possible to prevent him or her from doing so?
Isn't it possible to disable future dates in the date picker and to disable dates in the "end" date picker that are anterior to the selected date in the "start" date picker?